lib/utils.js

```diff
@@ -134,40 +134,91 @@ module.exports = {
   },
 
   /**
+   *
+   * References:
+   * - https://github.com/facebook/react-native/blob/main/Libraries/Utilities/Platform.ios.js
+   * - https://github.com/facebook/react-native/blob/main/Libraries/Utilities/NativePlatformConstantsAndroid.js
+   *
    * @returns An object with names and versions of the various components making up the context.
    */
   getVersions() {
     const packageJson = require("../package.json");
-    const packageVersion = packageJson.version;
+    const sdkVersion = packageJson.version;
     const environment = this.getEnvironment();
 
+    const sdk = "JS";
+
     try {
       if (environment === "reactnative") {
         const { Platform } = require("react-native");
+        let deviceName = "unknown";
+        let deviceVersion = "unknown";
+        if (Platform.OS === "ios") {
+          if (Platform.isPad()) {
+            deviceName = "iPad";
+          } else if (Platform.isTV()) {
+            deviceName = "AppleTV";
+          } else {
+            // RN doesn't support Apple Watch
+            deviceName = "iPhone";
+          }
+          deviceVersion = "unknown";
```

Comment:
You should be able to use `osVersion` here:

```suggestion
          deviceVersion = Platform.osVersion;
```

Reply:
I haven't found a way to tell me if it is an iPhone 10 or iPhone 11. `Platform.osVersion` will tell me the version of iOS or Android which is unrelated to the device.